### ISSUE TITLE
Can't type chars involving option key on macOS anymore

### ISSUE THREAD

**Issue by mrolli** [wez/wezterm#3866]
### What Operating System(s) are you seeing this problem on?

macOS

### WezTerm version

20230612-063953-baf9d970

### Did you try the latest nightly build to see if the issue is better (or worse!) than your current version?

Yes, and I updated the version box above to show the version of the nightly that I tried

### Describe the bug

I just realized that I can't type any chars from the <option> +<other_key> plane anymore, e.g. chars like ß (left option + s) or ∂ (left option + d). For ß (left option + s) something occurs that feels like a backspace, for other key cominations other things "happen" but never only a character. It reminds more of som

Not sure since when I've had this problem though. This issue is unrelated to my config as it occurs even if removing my config and running on defaults.
In Terminal.app and iTerm2 this works as expected and it used to work in Wezterm too. No recent changes on my machine except for the OS update to 13.4 although I also see this behaviour on my other MacBook still running on 13.3.1 (a).

What else can I provide to investigate this further?

### Configuration

no config

### Expected Behavior

Pressing left option + s should should print the char ß.

**Comment by ndevenish** [comment]
I have also encountered this while attempting to move to WezTerm; None of the characters listed in this post work, but also # (option-3) doesn't work, which is a pretty important character when writing bash and python!

I tried nightly, and also back to 20220101, which appeared to have the same issue. Currently running Ventura 13.2.1 (22D68).

**Comment by mrolli** [comment]
I was still not lucky so far. It somehow reminds me of something similar in nvim where the LHS of a keymap `<M-3>` is not same as `<A-3>` on macOS AKA the alt key mode is not the same the option key. But in the documentation of Wezterm I haven't seen something that outlines differences between alt and option.

In the karabiner eventviewer I see the correct keys being pressed and in case of iTerm2 and Terminal it does the right thing, i.e. print some greek/northern chars, but not in Wezterm, but it worked at some point.

**Comment by ndevenish** [comment]
Yes, this solves the issue for me. Confusingly, when searching tickets I found old references to setting `use_ime=true` to make this work.

This also isn't listed in the "Full" config reference page on https://wezfurlong.org/wezterm/config/lua/config/index.html, which only now do I notice says "This is not a complete list"!

**Comment by wez** [comment]
Let's re-frame this as a doc issue to get those added to the main list

**Comment by blixenkrone** [comment]
Hope it's ok to borrow the thread for this question:
I'm coming from Alacritty, and have been using a config setting that makes the option key behave as alt.

This config gives me the choice of either, but not both, which is a requirement for me to use WezTerm with Helix and Zellij, ie to switch focus and use key layers. 
Is there a way to configure this?
